testgrid/config.yaml

```diff
@@ -3483,6 +3501,24 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: cnx+e2e@vmware.com
 
+- name: vmware-conformance-cloud-provider
```

Comment:
Hi Ben,

Sorry about that. I should have clarified in the description. We have dashboards for the tests, and those follow the convention. These are for representing the same test results under the vmware category. You said this was possible since dashboards can be created multiple times for the same results. You also said something under a top-level category should be prefixed with that category. That’s why the new dashboards have the prefix “vmware-“.

I hope that helps to clarify the intent. Is this still a valid config?

Reply:
ah thanks, unburying a large backlog currently and skimmed this too quickly